README.md

```diff
@@ -43,6 +43,36 @@ passport.use(new SamlStrategy(
 );
 ```
 
+### Configure strategy for multiple providers
+
+You can pass a `getSamlOptions` parameter to `MultiSamlStrategy` which will be called before the SAML flows. Passport-SAML will pass in the request object so you can decide which configuation is appropriate.
+
+```javascript
+var MultiSamlStrategy = require('passport-saml').MultiSamlStrategy;
```

Comment:
Thanks for the update. these docs need to be updated to match as well. Hopefully we can make:

```
const MultiSamlStrategy = require( 'passport-saml/MultiSamlStrategy' );
```

And not this uglier and more confusing syntax:

```
const MultiSamlStrategy = require( 'passport-saml/lib/passport-saml/MultiSamlStrategy' );
```

This might involve moving the MultiSamlStrategy.js file, but I haven't tested.